action.yml

```diff
@@ -135,11 +140,22 @@ runs:
 
     - name: Download Product Zip Artifact
       uses: actions/download-artifact@v3
+      #if: ${{ inputs.do_zip_extract_step == true }}
+      if: ${{ inputs.do_zip_extract_step  == 'true'  && !contains(inputs.zip_name, '\n') }}
```

Comment:
🤔  I wonder if it would be better to let `actions/download-artifact` fail here with ex: `No such artifact: artifact_one.zip\nartifact2.zip`, rather than have a later `no such zip` error?

Reply:
See next comment, but this is cruft.